app/helpers/payment_details_helper.rb

```diff
@@ -19,33 +22,37 @@ def link_to_payment_details(text, options = {})
   end
 
   def payment_details_path(merchant_account = site_account, url_params = {})
-    return "" if ["bogus" ,""].include?(merchant_account.payment_gateway_type.to_s)
-    named_route = [:admin, :account, merchant_account.payment_gateway_type]
+    payment_gateway_type = merchant_account.payment_gateway_type
+    return "" if ["bogus",""].include?(payment_gateway_type.to_s)
+
+    named_route = [:admin, :account, payment_gateway_type]
     polymorphic_path(named_route, url_params)
```

Comment:
Variable assignment for a single use. We could remove it.
```suggestion
    polymorphic_path([:admin, :account, payment_gateway_type], url_params)
```